content/concepts/tokens/authorized-trust-lines.md

```diff
@@ -1,36 +1,41 @@
-3. The gateway's issuing address sends a TrustSet transaction authorizing the customer's trust line.
+The transaction to authorize a trust line must be signed by the issuing address, which unfortunately means an increased risk exposure for that address.
 
-**Tip:** An issuing gateway can authorize a trust line preemptively (step 3), before the customer has created it. This creates a trust line with zero limit, so that the customer's TrustSet transaction (step 2) sets the limit on the pre-authorized trust line. _(Added by the [TrustSetAuth amendment][].)_
+**Caution:** You can only enable `RequireAuth` if your account has no trust lines and no Offers in the XRP Ledger, so you must decide whether or not to use it _before_ you start issuing tokens.
 
-## RequireAuth Setting
+## With Stablecoin Issuing
 
-<!-- SPELLING_IGNORE: requireauth -->
+With a stablecoin on the XRP Ledger and use Authorized Trust Lines, the process of onboarding a new customer might look something like the following:
 
-The `RequireAuth` setting prevents all counterparties from holding balances issued by an address unless the issuing address has specifically approved a trust line with that counterparty for the currency in question.
+1. The customer registers with the stablecoin issuer's systems and sends proof of their identity (also known as "Know Your Customer", or KYC, information).
+2. The customer and stablecoin issuer tell each other their XRP Ledger addresses.
+3. The customer sends a [TrustSet transaction][] to create a trust line to the issuer's address, with a positive limit.
+4. The issuer sends a TrustSet transaction to authorize the customer's trust line.
 
-As a precaution, Ripple recommends that issuing gateways always enable `RequireAuth` on [operational addresses and standby addresses](issuing-and-operational-addresses.html), and then never approve any trust lines to those addresses. This prevents operational addresses and standby addresses from issuing currency in the XRP Ledger even by accident. This is a purely precautionary measure, and does not stop those addresses from transferring issued currency balances created by the issuing address, as they are intended to do.
+**Tip:** The issuer can authorize a trust line preemptively (step 3), before the customer has created it. This creates a trust line with zero limit, so that the customer's TrustSet transaction (step 2) sets the limit on the pre-authorized trust line. _(Added by the [TrustSetAuth amendment][].)_
 
-To use the Authorized Trust Lines feature, an issuer must also enable `RequireAuth` on its issuing address. After doing so, the issuing address must [submit a `TrustSet` transaction to approve each trust line](#authorizing-trust-lines) from a customer.
+## As a Precaution
```

Comment:
Make this a **Caution** or **Important** note instead of a sub-section.

Reply:
I disagree. This header exists to explain a non-intuitive use case for the feature, not to provide a warning to people. Also, having it as a heading means it's an anchor that you can link to directly.